bookish/Services/list_services.py

```diff
@@ -0,0 +1,47 @@
+from bookish.models import db
+from bookish.models.users import Users
+from bookish.models.books import Books
+
+
+def list_all_users():
+    try:
+        data = Users.query.all()
+    except Exception as e:
+        return {"status": "Error", "error_message": str(e)}
+
+    return [user.serialize() for user in data]

```

Comment:
I think want to return the data unserialized from the services as it's the controllers' job to prepare the response. 
Also use `jsonify` as it's Flask's recommended way of returning the responses.